### ISSUE TITLE
Improve Garbage Classification Model

### ISSUE THREAD

**Issue by vartika1801** [AditixAnand/Garbage_Classification#18]
Hi! I’d like to contribute by enhancing the model’s accuracy through:
Data augmentation 
Hyperparameter tuning (batch size, learning rate, optimizer)

**Comment by AditixAnand** [comment]
@kavyakhapra Hey Kavya! 👋
I've raised an issue for building a full-fledged website for our garbage classification model.
Feel free to suggest an alternative approach or idea if you have something better in mind!

**Comment by Nikhil210206** [comment]
@Mohitnegi56 please explain in detail

**Comment by Nikhil210206** [comment]
@vartika1801 any update on your work ?

**Comment by vartika1801** [comment]
I’ve drafted a standalone script that implements these changes and will run it on the dataset next. I’ll share results once it’s trained.

**Comment by vartika1801** [comment]
I’ve trained the updated model.
Training Accuracy: 98.07%
Validation Accuracy: 88.67%
Would it be okay if I go ahead and create a PR with this updated model?

**Comment by vartika1801** [comment]
created a PR

**Comment by Nikhil210206** [comment]
I will check @vartika1801
